web-components/src/wc_scss/themes/global--mdv2.scss

```diff
@@ -1,4 +1,86 @@
 @mixin global-theme-specfic-vars {
   --rem-base: 14px;
   font-size: var(--rem-base);
+
+  --wc-menu-padding: 0.5rem;
+  --wc-menu-item-height: 2.5rem;
+  --wc-menu-item-rounded-border-radius: 0.5rem;
+
+  //Override the light and dark theme colors vars
+  --main-bg-color: #{$mds-color-theme-background-solid-primary-normal};
+  --md-default-focus-outline-color: #{$mds-color-theme-outline-theme-normal};
+
+  // New Backgrounds
+  --md-primary-bg-color: #{$mds-color-theme-background-solid-primary-normal};
+  --md-primary-one-bg-color: #{$mds-color-theme-background-solid-secondary-normal};
+  --md-primary-two-bg-color: #{$mds-color-theme-background-solid-tertiary-normal};
+
+  --md-secondary-bg-color: #{$mds-color-theme-background-solid-secondary-normal};
+  --md-secondary-one-bg-color: #{$mds-color-theme-background-secondary-normal};
+  --md-secondary-two-bg-color: #{$mds-color-theme-background-secondary-hover};
+  --md-secondary-three-bg-color: #{$mds-color-theme-background-secondary-active};
+
+  --md-tertiary-bg-color: #{$mds-color-theme-background-solid-tertiary-normal};
+  // --md-tertiary-one-bg-color: #{$md-gray-95};
```

Comment:
Unused comments, more below

Reply:
Left to follow up and update in another PR. Need to find out what the appropriate mapping is but i think having so many updated is worth committing